src/nanoemoji/bitmap_tables.py

```diff
@@ -118,17 +153,15 @@ def _cbdt_bitmapdata_offsets(
 
 
 def _cbdt_bitmap_data(
-    config: FontConfig, metrics: BitmapMetrics, image_data: bytes
+    config: FontConfig, metrics: BitmapMetrics, image_data: PNG
 ) -> CbdtBitmapFormat17:
 
     bitmap_data = CbdtBitmapFormat17(b"", None)
     bitmap_data.metrics = SmallGlyphMetrics()
-    bitmap_data.metrics.height = config.bitmap_resolution
-    bitmap_data.metrics.width = config.bitmap_resolution
-    # center within advance
+    bitmap_data.metrics.width, bitmap_data.metrics.height = image_data.size
     bitmap_data.metrics.BearingX = metrics.x_offset
     bitmap_data.metrics.BearingY = metrics.y_offset
-    bitmap_data.metrics.Advance = _width_in_pixels(config)
+    bitmap_data.metrics.Advance = bitmap_data.metrics.width
```

Comment:
this is wrong, we know the advance width in pixel, it's `_width_in_pixels`; the `bitmap_data.metrics.width` is "the number of columns of data", i.e. the pixel width of the actual bitmap, the advance can be larger (and in fact it is larger in our default config which mimics noto-emoji's).

```suggestion
    bitmap_data.metrics.Advance = _width_in_pixels(config, image_data)
```

Reply:
the first frame is colrv1, the second is cbdt (from `main` branch, with correct advance) and the third frame is cbdt from this PR branch (with advance too narrow):

https://user-images.githubusercontent.com/6939968/161588016-51a410c4-3f92-4dbd-b3b7-8134e567a3c8.mov

Reply:
Bleh good catch, ty.